### ISSUE TITLE
Postgres example/docs

### ISSUE THREAD

**Issue by agustif** [accounts-js/accounts#682]
I made a shitty PR on #654 

I re-worked it separating only the example concerning the postgres/graphql/typeorm server.

Here's the repo link.
https://github.com/agustif/typeorm-server-typescript

I added necessary dependencies, @birkir did all the hard work building all the pieces necessary for this to work, and also open sourcing a great example of its use in Prime.

I just used that, and tried to keep it simple stupid.

Added a dockerfile for postgres, even if I just my mac's brew installed postgres instance, anyway...

Cleaning it up a bit further, updating and removing unnecessary dependencies.

Any kind of feedback would be appreciated, I could re-attempt a better PR after it if you think it would add value to the project.

Thanks for creating AccountsJS, awesome stuff!

**Comment by agustif** [comment]
Since I'm at it, I might recycle this and add a proper graphql-modules implementation for example, so you might not encounter the same problems as of I when just adding accounts as a last sideLoaded Module, not required by the app.module itself, which makes other modules loaded in app's module, not aware of context.user

**Comment by agustif** [comment]
I've gone ahead and opened https://github.com/accounts-js/accounts/pull/689, I tried to clean up as much stuff as possible that didn't make sense and only polluted the example, let me know if something else needs trimming or fixing to make it to master.

I've totally discarded all the special stuff @birkir had for PrimeCMS in place to implement both SSL when deploying to Amazon AWS in production, and also some fix with timezones which required some extra deps. I tried and it runs without it, I might want to document/give a notice that this might be a problem for some users, and that they should go look into prime-core to see how to intelligently solve it themselves, without having the code living in the example itself.
